chapters.md

```diff
@@ -189,6 +189,28 @@ Chapter 2                | false              | no
 ++Nested Chapter 2.2.1   | true               | no
 ++Nested Chapter 2.2.2   | false              | no
 
+### ChapterSegmentUID
+
+
+The `ChapterSegmentUID` is a binary value with a size of 128 bits and the base element to set up a `Linked Chapter` in 2 variations. For both variations the following 3 conditions MUST be met.
+
+ 1. The `EditionFlagOrdered Flag` is set to true
```

Comment:
You could use anchors to the definition of the element: `#editionflagordered-element`